### PR TITLE
Add new info

### PR DESCRIPTION
**Pull request by max21910**
Add :
-New CDN endpoint
-New Messaging endpoint
-Spotify Login Url

### CODE REVIEW

File: docs/domains/sub-domains.md [hunk 1]
```diff
@@ -69,6 +69,9 @@ title: Sub Domains
 |-------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
 | `cdn.bereal.network`          | New CDN running on `MCI Communications Services, Inc. d/b/a Verizon Business`   pointing to `storage.bere.al` Bucket               |
 | `cdn-resize.bereal.network`   | Image optimization CDN																											 |
+| `cdn-c.bereal.network`        | New CDN      																													     |
+| `cdn-goo.bereal.com`          | New CDN      																												         |
+| `cdn-c.bereal.network`        | New CDN 																												             |
```
Comment (userbradley) [inline docs/domains/sub-domains.md:74]: Lines `72` and `74` are repeated

**Review by userbradley — CHANGES_REQUESTED** [review]
Thanks as always, just one line seems to be repeated however!

File: docs/domains/sub-domains.md (hunk 1 above)
Comment (max21910) [reply docs/domains/sub-domains.md:74]: yes i will change that also i wanted to add this :
`## moments

## Verified Endpoints

| URL                                                             | Use           	 						  |                                                                                                                                         
|-----------------------------------------------------------------|-------------------------------------------|
| `https://mobile.bereal.com/api/bereal/moments/last/europe-west` | Gets last (eu) Bereal notification time   |

## Requests and Responses

Response

```json
{
		"id": {
			"NywegFScSAG83Fg4rugyI",
		    "startDate":"2024-01-14T10:37:05.108Z",
			"endDate":"2024-01-14T10:39:05.108Z",
			"region":"europe-west",
			"timezone":"Europe/Paris",
			"localTime":"11:37",
			"localDate":"2024-01-14"
}
```
`
but every-times the Github build fails any ideas ?

Comment (userbradley) [reply docs/domains/sub-domains.md]: I think the pre-commit action has a panicks. May be worth me having a look when I'm back from the gym.

Worst case we just push it and deal with it another day

Comment (max21910) [reply docs/domains/sub-domains.md]: yes we can push it if you are okay i will add this later on another PR :)

Comment (userbradley) [reply docs/domains/sub-domains.md]: Leave it as is, I'll fix the build step and merge it when I'm back

Thanks as always

Comment (max21910) [reply docs/domains/sub-domains.md]: Thanks, and sure, it's always a pleasure to search and add information to this repository. I learn so much.

**Pull request merged**